app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<h2>Sign up</h2>
+
+<div class="auth">
+
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email" %>
+    </div>
+
+    <div class="field">
+      <%= f.text_field :name, autocomplete: "name", placeholder: "Name"%>
+    </div>
+
+    <div class="field">
+      <%= f.password_field :password, autocomplete: false, placeholder: "Password"%>
+      <% if @minimum_password_length %>
+      <p class="min">
+        (<%= @minimum_password_length %> characters minimum)
+      </p>
+      <% end %>
+    </div>
+
+    <div class="field">
+      <%= f.password_field :password_confirmation, autocomplete: false, placeholder: "Confirm Password" %>
+    </div>
+
+    <div class="actions button">
+      <%= f.submit "Sign up" %>
+    </div>
+  <% end %>
+
+</div>
+
+<%= render "devise/shared/links" %>
+
+<%= link_to "Back", :back, class: "button" %>
```

Comment:
- [ ] Kindly increase the width of the form and submit button to resemble the wireframe. I also think it would be better to center the log in and receive confirmations links to follow the form position, instead of being being off and aligned to the left.

  ![image](https://github.com/Arch-Noize/rorblog/assets/81468675/ec183a13-d63a-4f2a-94fa-bd62bb32221b)

  ![image](https://github.com/Arch-Noize/rorblog/assets/81468675/a71342c4-d666-457e-9d78-cfdcc460c533)